### PR TITLE
cmds/core/kill: Create an unused port for testing kill

### PR DESCRIPTION
**Pull request by MedicineYeh**
The kill test might fail because PID 9999 exists. This commit create a PID that will be unused temporarily for testing purpose.

Signed-off-by: Medicine Yeh <medicinehy@gmail.com>

### CODE REVIEW

**Review by rminnich — APPROVED** [review]
Best. Bug. Report. Ever. 

There's still a potential race here but it's too small to worry about (I hope!)

**Review by rminnich — APPROVED** [review]

**Comment by 10000TB** [comment]
+1 Good stuff!

I can tell ya TestKillProcess has been failing a lot. Pulling this in. Hopefully, we never see this test fail in cbuilds.

**Comment by MedicineYeh** [comment]
Yeah, the potential race should be almost impossible since the unit test does not run more than a few seconds. Thanks for reviewing and merging.

File: cmds/core/kill/kill_test.go [hunk 1]
```diff
@@ -7,10 +7,19 @@ package main
...
 	"bytes"
 	"fmt"
+	"os/exec"
 	"strings"
 	"testing"
 )
 
+func getUnusedPID() string {
+	cmd := exec.Command("true")
+	cmd.Start()
```
Comment (abrender) [inline cmds/core/kill/kill_test.go:17]: Start() returns an error and it should probably be checked (along with changing getUnusedPID to be a [test helper](production/network/regtest:multidiff_flipper) which helps clarify where the failing call is:

```
func getUnusedPID(t *testing.T) {
  t.Helper()
  ...
  if err := cmd.Start(); err != nil {
    t.Fatalf("failed to start `true` command: %v", err) 
  }
}
```

**Pull request merged**